doc/pages/commands.asciidoc

```diff
@@ -299,6 +299,15 @@ but not really useful in that context.
 *debug* {info,buffers,options,memory,shared-strings,profile-hash-maps,faces,mappings}::
     print some debug information in the *\*debug** buffer
 
+== Module commands
+
+*provide-module* [<switches>] <name> <commands>::
+    declares a new module that is provided by the given commands. The commands
+    passed into the module are guaranteed to be run only once.
+
+*require-module* <name>::
+    guarantees the <name> module is loaded before continuing command execution
```

Comment:
"loading" a module isn't really a concept mentioned in the `provide-module` docs; how about:

> guarantees the commands associated with `provide-module <name>` have been evaluated before continuing command execution. Fails if no `provide-module <name>` command has been evaluated so far.